rjc_account_form/static/scss/style_account_report.scss

```diff
@@ -59,3 +63,31 @@
         }
     }
 }
+//style wht
+.o_report_withholding_tax_cert{
+    .payment_transaction_name{
+        margin-left: 675px;
+        position: fixed;
+        top: 1412px;
+    }
+    .value_date_day{
+        margin-left: 660px;
+        position: fixed;
+        top: 1435px;
+    }
+    .value_date_month{
+        margin-left: 750px;
+        position: fixed;
+        top: 1435px;
+    }
+    .value_date_year{
+        margin-left: 840px;
+        position: fixed;
+        top: 1435px;
+    }
+}
+//head_receipt_voucher
+.print_head {
```

Comment:
please try
`.header {
    .print_head{
        // your code
    }
}`
it's look good more than out style

Reply:
Ok, I can try.